app/src/main/java/com/github/se/cyrcle/ui/profile/ViewProfileScreen.kt

```diff
@@ -163,15 +174,26 @@ fun ViewProfileScreen(
                 })
           } else {
             DisplayProfileComponent(userState) {
+              // Edit Button directly in the Box
               Button(
                   text = stringResource(R.string.view_profile_screen_modify_profile_button),
                   onClick = { isEditing = true },
                   colorLevel = ColorLevel.TERTIARY,
                   testTag = "EditButton")
 
-              Spacer(modifier = Modifier.height(24.dp))
+              // Scrollable Column instead of LazyColumn
+              Column(
+                  modifier =
+                      Modifier.fillMaxWidth()
+                          .verticalScroll(rememberScrollState())
+                          .padding(top = 56.dp) // Adjust based on button height
+                          .testTag("ProfileContentSections")) {
+                    FavoriteParkingsSection(userViewModel, parkingViewModel, navigationActions)
+
+                    Spacer(modifier = Modifier.height(16.dp))
 
-              FavoriteParkingsSection(userViewModel, parkingViewModel, navigationActions)
```

Comment:
Careful because I think you also accidentally deleted `FavoriteParkingsSection.

Reply:
no it's ok, before there was only this. now it's a column with favoriteparkingssection/spacer/userreviewssection

Reply:
Sorry i didn't see it.... My bad